internal/dataplane/translator/ingressrules.go

```diff
@@ -21,7 +21,8 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/v3/internal/util"
 )
 
-func getClientCertIncompatibleProtocols() []string {
+// getNonTLSProtocols returns a list of protocols that do not make Kong establish a TLS connection with an upstream.
+func getNonTLSProtocols() []string {
```

Comment:
The intention was not to allow a global to be mistakenly modified. Since it cannot be defined as a constant, returning from a function simulates that.